src/main/webapp/app/shared/model/geocoding-result.model.ts

```diff
@@ -0,0 +1,8 @@
+export interface IGeocodingResult {
+  id?: number;
```

Comment:
would it be number or UUID?

Reply:
number I think, as we're using it everywhere and it works fine